resources/js/components/ui/Table/CellRenderer.tsx

```diff
@@ -0,0 +1,22 @@
+import {JSX, Show} from "solid-js";
+import {Dynamic} from "solid-js/web";
+
+export interface CellRendererProps<TProps extends object> {
+  /** The cell content (e.g. component/string) render */
+  component: any;
```

Comment:
Wyłączyłem linta no-any żeby był mniej uciążliwy tam gdzie trzeba użyć any, ale nadal oczywiście należy go używać tylko tam gdzie trzeba. A więc konkretnie: 1. w implementacji, gdy bardzo ułatwia albo umożliwia jakiś trik w przeciwnym razie niemożliwy (większość istniejących użyć w naszym kodzie) 2. w interface tylko wtedy, gdy to naprawdę jest any (na przykład parametr do console.log jest naprawdę any). Tu nie zachodzi żaden z tych przypadków. Na przykład bardzo nie chcemy dostać component:5 albo innej bzdury tego typu.

Zresztą w obecnym typowaniu TProps jest całkiem bez sensu, równie dobrze można go zastąpić bezpośrednio object. Co innego gdyby component było typu Component<TProps>, wtedy by to miało sens. Możesz tak spróbować zrobić, ale podejrzewam że łatwiej będzie otypować to odwrotnie - zobacz jak otypowany jest Dynamic i zrób tak samo. Pewnie użycia tego CellRenderer będą miały jakiś problem, wtedy można dodać odpowiednio |string, |undefined, czy co tam będzie trzeba. No i w implementacji CellRenderer na pewno będą potrzebne jakieś as any, chociażby w fallbacku.

Reply:
Hm, i do tego o ile widzę można je zmienić na unknown, przynajmniej tak się wydaje. Dobra, zajmę się tym przy okazji.

Reply:
Nie no, jednak nie tak łatwo. A poza tym było więcej any, jakoś w FelteForm niedawno było jeszcze, ale chyba przypadkiem usunąłem :) W każdym razie faktycznie włączę z powrotem tego linta.

Reply:
poprawione